ietf/group/views.py

```diff
@@ -1354,6 +1354,85 @@ def group_menu_data(request):
     return JsonResponse(groups_by_parent)
 
 
+@cache_control(public=True, max_age=30 * 60)
+@cache_page(30 * 60)
+def group_stats_data(request, years="3", only_active="1"):
```

Comment:
This signature, and the url dispatch to it, need some tweaking.

```
  <title>OverflowError
          at /group/groupstats.json/3333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333/1</title>
```

Using an int for a bool like this (and leaving the other values unguarded is probably not a good idea.
If we really want these parameters in the URL, the signature should accept a bool, and the URL dispatcher be configured to send True or False based on something like (ignoring years for the moment):
```
/group/groupstats.json/ ends up with True being passed
/group/groupstats.json/active/ ends up with True being passed.
/group/groupstats.json/all/ ends up with False being passed.
```
But shouldn't this be modeled more as a json API where you send years and only_active in a POST?

Reply:
Alternatively, do these _really_ need to be variants, or could they just be constants in the code now that you've explored the output some?

Reply:
I couldn't figure out how to pass arguments to template tags via URLs, maybe there is a better way?